components/purchase_modal/purchase.scss

```diff
@@ -298,24 +149,156 @@
+                        font-size: 14px;
+                        font-weight: 700;
+                        line-height: 14px;
+
+                        &.grayed {
+                            background: rgba(63, 67, 80, 0.08);
+                            color: rgba(63, 67, 80, 0.32);
+                        }
+
+                        &.active {
+                            border: 1px solid #1c58d9;
+                        }
+
+                        &.special {
+                            background: #1c58d9;
+                            color: #fff;
+                        }
+                    }
+
+                    .plan_price_rate_section {
+                        width: 100%;
+                        height: 150px;
+                        border-bottom: 1px solid #dadfe5;
+                        text-align: center;
+
+                        h4 {
+                            color: rgba(33, 63, 107, 0.8);
+                            font-family: 'Metropolis';
+                            font-size: 20px;
+                            font-style: normal;
+                            font-weight: 400;
+                            letter-spacing: -0.02em;
+                            line-height: 30px;
+                        }
+
+                        h1 {
+                            color: #1e325c;
+                            font-family: 'Metropolis';
+                            font-size: 52px;
+                            font-style: normal;
+                            font-weight: 700;
+                            letter-spacing: -0.02em;
+                            line-height: 60px;
+                        }
+
+                        .enterprise_price {
+                            font-size: 32px;
+                        }
+
+                        p {
+                            color: #152234;
+                            font-family: 'Metropolis';
+                            font-size: 18px;
+                            font-style: normal;
+                            font-weight: 400;
+                            letter-spacing: -0.02em;
+                            line-height: 20px;
+                        }
+                    }
+
+                    .plan_payment_commencement {
+                        margin-top: 16px;
+                        margin-bottom: 24px;
+                        color: #3f4350;
+                        font-family: 'Open Sans';
+                        font-size: 14px;
+                        font-style: normal;
+                        font-weight: 400;
+                        line-height: 20px;
+                    }
+
+                    .plan_billing_cycle {
+                        margin-top: 24px;
+                        color: rgba(63, 67, 80, 0.72);
+                        font-family: 'Open Sans';
+                        font-size: 12px;
+                        font-style: normal;
+                        font-weight: 400;
```

Comment:
Not sure if `font-weight` and `font-style` need to be overridden to `400` and `normal` in most places, either.